docs/build.mdx

```diff
@@ -23,13 +23,13 @@ Apktool is a collection of 1 project, containing subprojects and a few dependenc
 We use gradle to build. First clone the repository using either
 [SSH](ssh://git@github.com:iBotPeaches/Apktool.git) or [HTTPS](https://github.com/iBotPeaches/Apktool.git).
 
-1. `cd Apktool`
+1. Navigate to the Apktool folder: `cd Apktool`
 2. For remaining steps use `./gradlew` for unix based systems or `gradlew.bat` for windows.
 3. `[./gradlew][gradlew.bat] build shadowJar` - Builds Apktool, including final binary.
 4. Optional (You may build a Proguard jar) `[./gradlew][gradlew.bat] build shadowJar proguard`
 
-After build completes you should have a jar file at:
-`./brut.apktool/apktool-cli/build/libs/apktool-xxxxx.jar`
+After build completes you should have a bunch of jar files built. The best one to use is:
+`./brut.apktool/apktool-cli/build/libs/apktool-cli-all.jar`
```

Comment:
This isn't technically the best one to use. The `cli-all` is the un-proguarded fatjar. The proguarded one is named after the version name so it becomes like `apktool-vXXXX`

Reply:
Oh I see.
Which folder do I find that in?